parm/atm/jcb-base.yaml.j2

```diff
@@ -94,6 +98,16 @@ atmosphere_obsbiasout_suffix: ".satbias.nc"
 atmosphere_obsbiascovout_prefix: "{{APREFIX}}"
 atmosphere_obsbiascovout_suffix: ".satbias_cov.nc"
 
+bias_files:
```

Comment:
Seems users will need to edit this file every time they add a new _sensor_satellite_ even though all radiance bias correction files are in the same tarball.  Is there a way to get around this?

Reply:
The alternative would be to place the name of the tarball in the observation template itself, for each observation. @CoryMartin-NOAA 's concern was that this would interfere with the JEDI YAML reader (see https://github.com/NOAA-EMC/jcb-gdas/pull/31#issuecomment-2400784464). I actually don't think this will be the case, but my issue is more that the observation template should contain the information JEDI needs to run, not information that G-W needs for staging.

Ultimately, since there will be at least one more tarball for other observations (namely aircraft), information about what tarball is associated with which observation is going to have to be stored in some list.

Reply:
I think, while @RussTreadon-NOAA makes a good point, I think this approach is the most explicit and gives us the most flexibility. For example, this means, if we decide in the future we want to group the varbc files by sensor or platform we could do so. I could see benefit in doing it by platform so that we can expect when new coefficients need to be spun up when a new platform comes online

Reply:
@DavidNew-NOAA and @CoryMartin-NOAA , thank you for your replies.  We can / will likely revisit this as we add aircraft bias correction and start cycling.